source/guide_wallabag.rst

```diff
@@ -54,87 +54,95 @@ Remove the placeholder html file from the html folder:
+* ``mailer_port:`` 587 - *we need to serve the special* :manual_anchor:`smtp port <mail-access.html#smtp>` *here*.
+* ``mailer_encryption:`` ssl - The parameter for the encryption.
+* ``mailer_auth_mode:`` login - The authenication mode.
+* ``secret:`` Type in any random string here, minimum 32 characters and do not keep the default string!
+* ``twofactor_sender:`` Choose an email address to be used as sender.
+* ``fosuser_registration:`` false - Set this to false, otherwise anyone can register at your wallabag instance
+* ``from_email:`` Choose an email address to be used as sender (can be the same as above).
 
 .. code-block:: console
- :emphasize-lines: 1,2,10,11,12,17,23,25,26,28,30,31,32,33
-
-  [isabell@stardust ~]$ cd ~/html
-  [isabell@stardust html]$ make install
-  [...]
-  Creating the "app/config/parameters.yml" file
-  Some parameters are missing. Please provide them.
-  database_driver (pdo_mysql):
-  database_driver_class (null):
-  database_host (127.0.0.1):
-  database_port (null):
-  database_name (wallabag): isabell_wallabag
-  database_user (root): isabell
-  database_password (null): MySuperSecretPassword
-  database_path (null):
-  database_table_prefix (wallabag_):
-  database_socket (null):
-  database_charset (utf8mb4):
-  domain_name ('https://your-wallabag-url-instance.com'): https://isabell.uber.space
-  mailer_transport (smtp):
-  mailer_host (127.0.0.1): 127.0.0.1:587
-  mailer_user (null): isabell@uber.space
-  mailer_password (null): MySuperSecretPassword
-  locale (en):
-  secret (ovmpmAWXRCabNlMgzlzFXDYmCFfzGv): *!!set.random.string!!*
-  twofactor_auth (true):
-  twofactor_sender (no-reply@wallabag.org): isabell@uber.space
-  fosuser_registration (true): false
-  fosuser_confirmation (true):
-  from_email (no-reply@wallabag.org): isabell@uber.space
-  [...]
-  * Would you like to create a new admin user (recommended)? (yes/no) [yes]: yes
-  * Username [wallabag]: admin
-  * Password [wallabag]: [hidden] *!!choose.your.own!!*
-  * Email []: isabell@uber.space
-  [...]
-  Config successfully setup.
-  [OK] Wallabag has been successfully installed.
-  [OK] You can now configure your web server, see https://doc.wallabag.org
-  [isabell@stardust html]$
-
-.. note:: You just need to set the highlighted values (as explained above), for the others just accept the default values with *enter*. You can change all the settings afterwards inside the ``~/html/app/config/parameters.yml`` file.
-
-To apply changes to parameters.yml, you have to clear your cache by deleting everything in ``~/var/cache`` with this command:
-
-``[isabell@stardust html]$ bin/console cache:clear -e=prod``.
-
-You still need to forward your document root to the ``/web`` folder where the public content is located. Therefore create a ``.htaccess`` file inside the ``~/html`` folder with the following content:
+  :emphasize-lines: 6,7,8,13,14,16,17,19,20,21,23,25,26,30
+# This file is auto-generated during the composer install
```

Comment:
All of these lines need to be indented by 2 spaces like the other code blocks, otherwise the formatting is messed up: https://deploy-preview-1535--uberlab.netlify.app/guide_wallabag/#installation-configuration

Reply:
I have made the adjustments.